pkg/models/testcase.go

```diff
@@ -4,11 +4,18 @@ type Kind string
 type BodyType string
 type Version string
 
-const (
-	V1Beta1 Version = Version("api.keploy.io/v1beta1")
-	V1Beta2 Version = Version("api.keploy.io/v1beta2")
+var (
+	V1Beta1 = Version("api.keploy.io/v1beta1")
```

Comment:
This should be a constant and to store the current version, an un-exported global scoped variable.

Reply:
Done